### PR TITLE
Feat/add guide video

### PR DESCRIPTION
**Pull request by iamvijaysaran**
Fixes #

### Checklist
- [x] My branch is up-to-date with upstream/main branch.
- [x] Everything works and tested for major version of Python/NodeJS/Go and above.
- [ ] I ran pre-commit checks against my changes.
- [ ] I've written tests against my changes and all the current present tests are passing.

### Current behaviour

After running ./build.sh, in step 6, our documentation advises the user to enter the Burner wallet address, which does not come up in the real process.
After signing the private key the next line contradicts the above statement

### New expected behaviour

After entering "yes" to assign Burner wallet, documentation advises the user to directly sign with the private key associated with the account holding the slot just like it happens in the terminal.
User is now informed to enter the private key used to mint the devnet slot to avoid confusion.

### CODE REVIEW

File: docs/build-with-powerloom/devnet/getting-started.md [hunk 1]
```diff
@@ -182,7 +182,6 @@ Do you want to assign a burner wallet to a slot? (yes/no):
 ```
 
 Enter yes and then enter your burner wallet address when prompted.
```
Comment (xadahiya) [inline docs/build-with-powerloom/devnet/getting-started.md:184]: We should remove this line, too; we never ask for the burner wallet, only the holding address private key.

Comment (iamvijaysaran) [reply docs/build-with-powerloom/devnet/getting-started.md]: Noted. Done, I raised a PR; please review it.

File: docs/build-with-powerloom/devnet/getting-started.md [hunk 2]
```diff
@@ -171,23 +175,22 @@ or
...
+#### Step 6: Assign your Wallet
 
 ```bash
-Do you want to assign a burner wallet to a slot? (yes/no):
+Do you want to assign a wallet to a slot? (yes/no):
 ```
 
-Enter yes and then enter your burner wallet address when prompted.
-- This is the `SIGNER_ACCOUNT_ADDRESS` that was set during [Step 3](#step-3-configure-the-environment-variables).
+Enter yes and then enter your wallet address when prompted.
```
Comment (xadahiya) [inline docs/build-with-powerloom/devnet/getting-started.md:184]: @iamvijaysaran, we never ask for any wallet address; you've run the node multiple times.

Comment (xadahiya) [inline docs/build-with-powerloom/devnet/getting-started.md:184]: Let's remove this line; we never ask the user for the burner wallet. It gets picked up from the `.env` file.

Comment (iamvijaysaran) [reply docs/build-with-powerloom/devnet/getting-started.md]: Noted. Raised a new PR : https://github.com/PowerLoom/docs/pull/37/commits/8e794eefe368ac01a5aa684a18ae0468f79a50f1

**Review by xadahiya — APPROVED** [review]
LGTM

**Review by kanikamishra — APPROVED** [review]
LGTM

**Review by anomit — APPROVED** [review]
LGTM

**Pull request merged**